spec/cellular/backends/cool_sms_spec.rb

```diff
@@ -14,6 +14,19 @@
     }
   }
 
+  let(:payload){
```

Comment:
Use spaces around `{` and `}` when used for blocks in line with https://github.com/bbatsov/ruby-style-guide